pkg/network/protocols/postgres/operations.go

```diff
@@ -29,6 +29,12 @@ const (
 	AlterTableOP
 	// TruncateTableOP represents a TRUNCATE operation.
 	TruncateTableOP
+	// ShowOP represents the 'SHOW' command to display the current setting of run-time parameters
+	ShowOP
+	// BeginOP represents the 'BEGIN' command to initiate a transaction
+	BeginOP
+	// CommitOP represents the 'COMMIT' command commits the transaction
+	CommitOP
```

Comment:
Why did you add this to the PR?

Reply:
These operations are used in Postgres queries, discovering these operations will reduce errors when operation was not found.

Reply:
This is outside the scope of this PR. We have Jira tickets to add support for these operations in the future. ([example](https://datadoghq.atlassian.net/browse/USMON-1185))

Reply:
Do you want me to do it first? this is source of undefined counters